java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/CommunicatorObserverI.java

```diff
@@ -2,6 +2,15 @@
 
 package com.zeroc.Ice;
 
+import com.zeroc.Ice.Instrumentation.CommunicatorObserver;
```

Comment:
I don't know why the formatter moved this around.

Reply:
Did gradle formatter updated this code, or was it Visual Studio Code?

Reply:
Hm... I think maybe it was VS Code first, then I disabled the setting, then used rewrite.